catalog/app/containers/Auth/SignUp.js

```diff
@@ -28,6 +29,48 @@ const Container = Layout.mkLayout('Complete sign-up')
 
 const MUTEX_ID = 'password'
 
+const useWeakPasswordIconStyles = M.makeStyles((t) => ({
+  icon: {
+    color: t.palette.warning.dark,
+  },
+}))
+
+function WeakPasswordIcon() {
+  const classes = useWeakPasswordIconStyles()
+  return (
+    <M.Tooltip title="Password is too weak">
+      <M.Icon className={classes.icon} fontSize="small" color="inherit">
+        error_outline
+      </M.Icon>
+    </M.Tooltip>
+  )
+}
+
+function PasswordField({ input, ...rest }) {
+  const { value } = input
+  const score = PasswordStrength.useScore(value)
+  const isWeak = score >= 0 && score <= 2
+  return (
+    <>
+      <Layout.Field
+        InputProps={{
+          endAdornment: isWeak && (
+            <M.InputAdornment position="end">
+              <WeakPasswordIcon />
+            </M.InputAdornment>
+          ),
+        }}
+        helperText="Hint: use combination of two-three words, mix uppercase with lowercase letters, and use special characters"
```

Comment:
shouldnt we use hints from `zxcvbn`?

```
result.feedback   # verbal feedback to help choose better passwords. set when score <= 2.

  result.feedback.warning     # explains what's wrong, eg. 'this is a top-10 common password'.
                              # not always set -- sometimes an empty string

  result.feedback.suggestions # a possibly-empty list of suggestions to help choose a less
                              # guessable password. eg. 'Add another word or two'
```

Reply:
> I use result.feedback.suggestions with the new changeset. But warning almost always is empty for me and shows some strange warnings

ok, you can revert if you think it's not useful (i trust your judgement)

Reply:
Suggestions are useful, warning for some reason - not useful :man_shrugging: